OpenShift/helm/nexus-repository-manager/Chart.yaml

```diff
@@ -1,8 +1,25 @@
-apiVersion: v1
-name: sonatype-nexus
-version: 0.2.0
-appVersion: 3.20.1-01
-description: Sonatype Nexus is an open source repository manager
+apiVersion: v2
+name: nexus-repository-manager
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 21.1.0
```

Comment:
can we match this up to the app version plus an incremental chart version, like we discussed for iq?

Reply:
sure